Gemfile

```diff
@@ -106,13 +107,12 @@ gem "staccato", "~> 0.5" # Ruby Google Analytics Measurement
 gem "sterile", "~> 1.0" # Transliterate Unicode and Latin1 text to 7-bit ASCII for URLs
 gem "stripe", "~> 5.55" # Ruby library for the Stripe API
 gem "strong_migrations", "~> 1.2" # Catch unsafe migrations
+gem "terser", "~> 1.1" # Terser is a Ruby wrapper for the terser JavaScript parser and minifier
```

Comment:
Terser replaces Uglifier because it supports ES6+ better. Ideally we should use esbuild completely, but that'll be for the future.